utils/eeprom_metadata.py

```diff
@@ -131,15 +147,34 @@ def getArguments():
     application_name = args['appname']
     output_file = args['output']
     print_header_result = args['verbose']
-    return binary_file, application_name, output_file, print_header_result
-
+    dont_auto_convert = args['dont_convert']
+    return binary_file, application_name, output_file, print_header_result, dont_auto_convert
+
+def check_if_binary(check):
+    file_name, file_ext = splitext(check)
+    if file_ext == ".bin":
+        return True, check
+    elif file_ext == ".hex":
+        if not dont_auto_convert:
+            # check=True is used to raise an exception if the command does not run successfully (doesn't exist, bad file, etc)
+            subrun(['avr-objcopy', '-I', 'ihex', binary_file, '-O', 'binary', file_name+'.bin'], check=True)
+            # Return the new file's path
```

Comment:
Superfluous, remove.

Reply:
If you are referring to the comment on line 161, that has been removed in the newest commit as well.